src/com/Creator.java

```diff
@@ -1,12 +1,39 @@
 package com;
+import java.util.Scanner;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
 
 public class Creator {
     public void createWorkOrders() {
         // read input, create work orders and write as json files
+
+        WorkOrder newWorkOrder = new WorkOrder();
+        Scanner input = new Scanner(System.in);
+
+        status = newWorkOrder.setStatus(Status.INITIAL);
+
+        System.out.println("Please enter your name.");
+        senderName = input.nextLine();
+
+        ObjectMapper mapper = newObjectMapper();
+        int workOrderIDNo = newWorkOrder.getId();
+        mapper.writeValue(new File(workOrderIDNo + ".json"),newWorkOrder);
+        String jsonString = mapper.writeValueAsString(newWorkOrder);
```

Comment:
This is just gonna make a string representing the JSON which isn't needed as the ling above is writing it to an actual file